.meta/dietpi-imager

```diff
@@ -243,7 +243,7 @@ Do you want to overwrite or backup the existing file to $OUTPUT_IMG_NAME.img.bak
 			truncate --size=$IMAGE_SIZE "$FP_SOURCE_IMG"
 
 			# Rename and move to /root, if source image != output image already
-			[[ /root/$OUTPUT_IMG_NAME.img != $FP_SOURCE_IMG ]] && mv "$FP_SOURCE_IMG" $OUTPUT_IMG_NAME.img
+			mv -n "$FP_SOURCE_IMG" $OUTPUT_IMG_NAME.img
```

Comment:
Are you sure that the no-clobber check is done prior to the "same file" check so that no error message appears? Generally it is a habbit of me to do such checks before calling any external command, for performance reasons, however increases code of course.

Reply:
Yes, suggested change is bogus. When the user has previously chosen to overwrite OUTPUT_IMG_NAME.img my code wont do that.

Reply:
Ah yes, was not even thinking about that.